.gitignore

```diff
@@ -103,3 +103,9 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# Django static
+/static
+
+# npm
+node_modules/
```

Comment:
Should it have the prepended slash like `/static` above?

Reply:
Yes.